main.py

```diff
@@ -1,8 +1,13 @@
 from bot.bot import MusicBot
 from bot.structures.config import Config
+import os
 
 if __name__ == "__main__":
-    cfg = Config.from_file("config.json")
+    cfg = Config.from_file("./config.json")
```

Comment:
Maybe check if an env var exists instead and select the config init based on that:

```py
if os.environment.get("FLUTE_PREFIX") is not None:
    # get from env
else:
    # get from file
```

Reply:
We'll just check if file exists for now.